packages/services/usage/src/usage.ts

```diff
@@ -268,10 +297,25 @@ export function createUsage(config: {
     status = newStatus;
   }
 
+  producer.on(producer.events.CONNECT, () => {
+    logger.info(`Kafka producer: connected`);
+
+    if (status === Status.Unhealthy) {
+      changeStatus(Status.Ready);
```

Comment:
My thought here is that we should be more proactive with setting the usage service to unhealthy and ready, instead of waiting for messages to fail.